tslint.json

```diff
@@ -10,6 +10,7 @@
         "no-internal-module": true,
         "no-trailing-whitespace": true,
         "no-var-keyword": true,
+        "no-unused-variable": true,
```

Comment:
I don't see it as a big problem if our project builds with unused variables, if we can trust that they are already being caught during linting & testing.

While it would be nice to check in both steps, practically we don't want to be maintaining two different linting rulesets.